its/pom.xml

```diff
@@ -31,7 +31,7 @@
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator</artifactId>
-      <version>3.35.0.2707</version>
```

Comment:
This update is not really needed but I took the opportunity to bring orchestrator up to date.